src/tests/unit/tests/DetailsView/components/save-assessment-button.test.tsx

```diff
@@ -50,100 +49,22 @@ describe('SaveAssessmentButton', () => {
-                expect(getMockComponentClassPropsForCall(Dialog, 2).hidden).toEqual(false);
-            });
-
-            it('dialog is hidden (dismissed) when onDismiss is called', () => {
-                act(() => {
-                    getMockComponentClassPropsForCall(Dialog, 2).onDismiss();
-                });
-                expect(getMockComponentClassPropsForCall(Dialog, 3).hidden).toEqual(true);
-            });
-        });
-        let wrapper: RenderResult;
-        describe('interaction', () => {
-            beforeEach(() => {
-                useOriginalReactElements('@fluentui/react', [
-                    'Dialog',
-                    'DialogFooter',
-                    'Stack',
-                    'Checkbox',
-                    'PrimaryButton',
-                    'ActionButton',
-                ]);
-
-                wrapper = render(<SaveAssessmentButton {...propsStub} />);
-                fireEvent.click(wrapper.getByRole('link'));
-            });
-
-            it('when "dont show again" box is clicked, set the showSaveAssessmentDialog user config state to `false`', () => {
-                // The "Don't show again" checkbox logic is inverted
-                const checkbox = wrapper.getByRole('checkbox');
-                // Check "Don't show again" = true
-
-                fireEvent.click(checkbox);
-                // showSaveAssessmentDialog = false ("Enable the dialog" = false)
-                userConfigMessageCreatorMock.verify(
-                    x => x.setSaveAssessmentDialogState(false),
-                    Times.atLeastOnce(),
-                );
-            });
-
-            it('should call saveAssessment on click', async () => {
-                assessmentActionMessageCreatorMock.verify(
-                    x => x.saveAssessment(It.isAny()),
-                    Times.atLeastOnce(),
-                );
-            });
-            it('dialog is hidden (dismissed) when "got it" button is clicked', async () => {
-                const gotItButtonProps = getMockComponentClassPropsForCall(PrimaryButton);
-                act(() => {
-                    gotItButtonProps.onClick();
-                });
-                // since upgrading to React 18, this test is nondeterministic between being run in
-                // isolation and being run with the entire test suite. If you are working on this test
-                // and running it all by itself locally, it will include an additional empty render after
-                // the link click in the beforeEach, making this test fail.
-                const getProps = getMockComponentClassPropsForCall(Dialog, 3);
-                expect(getProps.hidden).toEqual(true);
-            });
-        });
+    it('should render per snapshot', async () => {
+        const wrapper = render(<SaveAssessmentButton {...propsStub} />);
+        expect(wrapper.asFragment()).toMatchSnapshot();
     });
 
-    describe('on dialog disabled', () => {
```

Comment:
this test is missing

Reply:
Hi @madalynrose  added now in save-assessment-dialog.test, please re-review.